examples/vite/src/App.vue

```diff
@@ -14,7 +14,7 @@ onErrorCaptured((err) => {
 })
 
 const metamask = window.ethereum
-const contract = new Contract(metamask)
+const contract = createContract(metamask)
```

Comment:
Why not new Contract() directly?

Reply:
Cuz TypeScript cannot support generic constructor https://github.com/microsoft/TypeScript/issues/40451

Like viem, it also used functions to create a instance instead of `new`.

Reply:
Think we should add `createIndexer` to meet consistency later.